modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -43,7 +30,31 @@ const getPlainTextContent = (editor: Editor, body: HTMLElement) => {
       Remove.unwrap(elem);
     }
   });
+};
+
+export const cleanupInputNames = (parent: SugarElement<Node>): void => {
+  const inputs = SelectorFilter.descendants(parent, 'input');
+
+  Arr.each(inputs, (input) => {
+    Attribute.remove(input, 'name');
```

Comment:
I haven't tested it yet but was wondering if an `id` attribute could cause similar issues of radio buttons being unselected?

Reply:
I did a check, this doesn't seem to be the case.